### ISSUE TITLE
[Feature Request] Add some basic examples 

### ISSUE THREAD

**Issue by lol219** [Soheab/vacefron.py#10]
Hello there ,hope you are fine .
Can you make some examples (qnippets basic stuff) to use it ,also some examples too if we want to use it in discord.py 
Thanks :D

**Comment by Soheab** [comment]
Yes, planning to provide examples in a folder called "examples".

Feel free to PR some example if you can.
